setup.py

```diff
@@ -102,6 +102,6 @@
                         'scikit-bio >= 0.2.3, < 0.3.0', 'wtforms == 2.0.1',
                         'qiime >= 1.9.0, < 1.10.0', 'moi >= 0.2.0, < 0.3.0',
                         'sphinx-bootstrap-theme', 'Sphinx >= 1.2.2',
-                        'gitpython'],
+                        'gitpython', 'seaborn'],
```

Comment:
Can you remove `seaborn` from here and add it to `qiita_plugins/biom_type/setup.py`? I think you will need to add pandas there too, as it is a new dependency for the plugin.